trino/serial.go

```diff
@@ -147,9 +148,8 @@ func Serial(v interface{}) (string, error) {
 	case string:
 		return "'" + strings.Replace(x, "'", "''", -1) + "'", nil
 
-		// TODO - []byte should probably be matched to 'VARBINARY' in trino
 	case []byte:
-		return "", UnsupportedArgError{"[]byte"}
+		return "X'" + strings.ToUpper(hex.EncodeToString(x)) + "'", nil
```

Comment:
This adds support for using `[]byte` in query params, right? What about reading VARBINARY columns in query results?

Also, please update the `README`, there's a section there with a list of supported and not supported data types.

Reply:
reading VARBINARY columns in query results we need to implement sql.Scanner interface. 
like 
`type TrinoVARBINARY []byte

func (h *TrinoVARBINARY) Scan(src any) (err error) {
	switch s := src.(type) {
	case string:
		var bytes []byte
		bytes, err = base64.StdEncoding.DecodeString(s)
		if err == nil {
			*h = bytes
		}
	default:
		err = fmt.Errorf("unsupported type: %T", src)
	}
	return
}
`

Reply:
Would to like to try in this PR? I know I'm asking to extend the scope of work, so it's also ok to do this in a follow up PR

Reply:
I will do this in next PR. I hope to also support driver.Valuer interface